CMSIS/NN/Tests/UnitTest/README.md

```diff
@@ -53,9 +53,9 @@ First clone the Arm Ethos-U Core Software and Arm Ethos-U Core Platform projects
     ```mkdir build```
     ```cd build```
     ```cmake .. -DCMAKE_TOOLCHAIN_FILE=</path/to/Ethos-u-core-platform>/cmake/toolchain/arm-none-eabi-gcc.cmake -DETHOSU_CORE_PLATFORM_PATH=</path/to/Ethos-u-core-platform> -DUSE_ETHOSU_CORE_PLATFORM=ON -DTARGET_CPU=cortex-m55 -DETHOS_U_CORE_SOFTWARE_PATH=</path/to/Ethos-u-core-software> -DCORE_SOFTWARE_ACCELERATOR=CMSIS-NN -DCORE_SOFTWARE_RTOS=None```
-    ```make test_arm_depthwise_conv_s8_opt```
+    ```make cmsis_nn_unit_tests```
```

Comment:
Let's have both lines as examples(test_arm_depthwise_conv_s8_opt and cmsis_nn_unit_tests ) so it is clear that one can run a targeted test or the entire suite..

Reply:
Ok